package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Tracks deployments of sample applications",
   "main": "app.js",
   "scripts": {
-    "postinstall": "./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
+    "postinstall": "bower install --allow-root && ./admin.js track && ./admin.js db put && ./admin.js ddoc put && ./admin.js clean repository_url_hash",
```

Comment:
OK, so the `--allow-root` is necessary for this to run correctly in Bluemix? In that case I guess it needs to stay. What about adding `--config.interactive=false`?